src/example-useScrollSync.tsx

```diff
@@ -0,0 +1,146 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+import { useScrollSync } from "react-scroll-sync";
+
+const App: React.FC = () => {
```

Comment:
No need to type it. It’s discouraged by React team afaik.

Reply:
Good point! Especially since `App` doesn't make use of `children`, which `React.FC` implies.

So I agree! 😄 

Out of curiosity, do you have a link to the React team discouraging this? I'm interested in how they phrased it.

Reply:
I can't remember where I read about it TBH.